<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qHomeClass</class>
 <widget class="QMainWindow" name="qHomeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qHome</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridCell1"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridCell2"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridCell3"/>
   </widget>
   <widget class="QLabel" name="wethInfo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="wethIcon"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridCell4"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridCell5"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qHome.qrc"/>
 </resources>
 <connections/>
</ui>
